docs/examples/ingestion/redis_ingestion_pipeline.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        "What documents do you see?"
</pre><p>it is good to know we can do this way to count documents</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Haha it's a little sneaky, since the file_name is showing up in the node text sent to the LLM

Alternatively, you can set the top k very high and just check response.source_nodes